### PR TITLE
html-converter-next links testing

### PR DESCRIPTION
**Pull request by redbow-kimee**

### CODE REVIEW

File: test/fixtures/semantic-html5-scenarios/autolink.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+Go to https://docs.asciidoctor.org to learn about asciidoctor.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/autolink.adoc:1]: nitpick:

```suggestion
Go to https://docs.asciidoctor.org to learn about Asciidoctor.
```

File: test/fixtures/semantic-html5-scenarios/autolink.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p>
+Go to <a href="https://docs.asciidoctor.org" class="bare">https://docs.asciidoctor.org</a> to learn about asciidoctor.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/autolink.html:2]:
```suggestion
Go to <a href="https://docs.asciidoctor.org" class="bare">https://docs.asciidoctor.org</a> to learn about Asciidoctor.
```

File: test/fixtures/semantic-html5-scenarios/mailto-links.adoc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Try a mailto link mailto:example@asciidoctor.org[].
+Email mailto:example@asciidoctor.org[Asciidoctor].
+Email asciidoctor with a subject/body mailto:example@asciidoctor.org[Asciidoctor, Hello, Body Text].
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/mailto-links.adoc:3]:
```suggestion
Email Asciidoctor with a subject/body mailto:example@asciidoctor.org[Asciidoctor, Hello, Body Text].
```

File: test/fixtures/semantic-html5-scenarios/mailto-links.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<p>
+Try a mailto link <a href="mailto:example@asciidoctor.org">example@asciidoctor.org</a>.
+Email <a href="mailto:example@asciidoctor.org">Asciidoctor</a>.
+Email asciidoctor with a subject/body <a href="mailto:example@asciidoctor.org?subject=Hello&amp;body=Body+Text">Asciidoctor</a>.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/mailto-links.html:4]:
```suggestion
Email Asciidoctor with a subject/body <a href="mailto:example@asciidoctor.org?subject=Hello&amp;body=Body+Text">Asciidoctor</a>.
```

File: test/fixtures/semantic-html5-scenarios/url-macro-link.adoc [hunk 5]
```diff
@@ -0,0 +1 @@
+Visit https://docs.asciidoctor.org[Asciidoctor Docs] to learn more about asciidoctor.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/url-macro-link.adoc:1]:
```suggestion
Visit https://docs.asciidoctor.org[Asciidoctor Docs] to learn more about Asciidoctor.
```

File: test/fixtures/semantic-html5-scenarios/url-macro-link.html [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<p>
+Visit <a href="https://docs.asciidoctor.org">Asciidoctor Docs</a> to learn more about asciidoctor.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/url-macro-link.html:2]:
```suggestion
Visit <a href="https://docs.asciidoctor.org">Asciidoctor Docs</a> to learn more about Asciidoctor.
```

**Comment by Mogztter** [comment]
Thanks!

**Comment by redbow-kimee** [comment]
suggestions accepted

**Pull request merged**